src/frontend/components/Note.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component, PropTypes } from 'react';
+import Radium from 'radium';
+import Paper from 'material-ui/lib/paper';
+import Colors from 'material-ui/lib/styles/colors';
+import {Element} from 'react-scroll'
+import CommentList from "./CommentList"
+
+const styles = {
+  rowContainer: {
+    display: 'flex',
+    flexFlow: 'row nowrap',
+    justifyContent: 'center',
+    alignItems: 'stretch',
+    overflow: 'hidden',
+    width: '100%'
+  },
+
+  columnContainer: {
+    display: 'flex',
+    flexFlow: 'column nowrap',
+    justifyContent: 'stretch',
+    alignItems: 'stretch'
+  },
+
+  notesColumn: {
+    flex: '0 1 60%',
+    boxShadow: '0 0px 0px rgba(0, 0, 0, 0.15)',
+  },
+
+  commentsColumn: {
+    flex: '0 1 35%',
+    boxShadow: '0 0px 0px rgba(0, 0, 0, 0.15)'
+  },
+
+  title: {
+    fontWeight: 100,
+    color: Colors.blueGrey400
+  },
+
+  note: {
+    fontWeight: 300,
+    fontSize:'0.9rem',
+    color: Colors.grey600,
+    lineHeight:'1.5rem'
+  },
+
+  list: {
+    listStyle: 'circle outside'
+  }
+};
+
+@Radium
+export default class Note extends React.Component {  
+
+  static propTypes = {
+    name: PropTypes.string.isRequired
+  };
+  render() {
+    const {name } = this.props;
+    return (
+      <Element name={name}>
```

Comment:
Ah, is this the preferred way of creating a generic top-level component? I had been putting <div> but it felt incorrect.